### DISCUSSION TITLE
Issues experienced by G6 Firefly users

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#1912] [Ideas]
#### Your environment
xDrip: May 21, 2021
Android: any
<br/>  

#### Expected behavior
No confusing, misleading, or unexpected logs, messages, menu options, or behavior
<br/>  

#### The following is a list of issues that could be observed by a G6 Firefly user.
- [x]  1- Temperature is not shown on the system status page.   https://github.com/NightscoutFoundation/xDrip/pull/1722
- [x]  2- Add Calibration is not an option.  There is only an initial calibration option.    https://github.com/NightscoutFoundation/xDrip/pull/1727
- [x] 3- The following message is observed after a calibration.  Got invalid impossible slope calibration!  
- [ ] 4- Multiple instances of the following log are observed after a calibration.  Received backfill packet out of sequence  
- [x] 5- Pop-up option to hard reset transmitter even though a Firefly cannot be reset
- [ ] 6- Failure to start sensor
- [ ] 7- The following message after calibrating using the treatment menu.  Cannot send calibration older than 1 hour.  

#### Summary
None of these issues is critical.
They can all be circumvented, or safely ignored.  However, for an inexperienced user, these can be very confusing or time consuming.
<br/>  

#### Steps to reproduce the behavior:
Use a Firefly G6, meaning a G6 transmitter with a firmware starting with 2.18 or 2.27.
<br/>  

#### There are open or closed issues related to some of these:  
https://github.com/NightscoutFoundation/xDrip/issues/1239
https://github.com/NightscoutFoundation/xDrip/issues/1415
https://github.com/NightscoutFoundation/xDrip/issues/1310
<br/>  

#### Plan  
I would like to address all of these.  However, it may take time.  We have to ensure nothing is affected negatively before any fix is accepted.

**Comment by tolot27** [comment]
@Navid200 Please can you reformat your issue text to use a [task list](https://github.github.com/gfm/#task-list-items-extension-) rather than the numbered list and append the related PR/issue to each list item? Then, we can simply click on the checkbox if a task has been completed. Thanks!

If you need help to address the remaining issues, please inform me at Gitter.
